js/crm.backdrop.js

```diff
@@ -0,0 +1,14 @@
+// http://civicrm.org/licensing
+CRM.$(function($) {
+  $(document).on('crmLoad', function(e) {
+    $('#admin-bar').css('display', 'none');
```

Comment:
For some reason I don't get the same results when using .show() and .hide(). I can't get it to hide the Backdrop admin bar on page load with .hide() and using .show() on the clicks seems to interfere with the page load as well.